### PR TITLE
release notes 21.1.0

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/21.1.0.md [hunk 1]
```diff
@@ -0,0 +1,1472 @@
...
+
+---
+
+### EWS Pack v1.6.1
+#### Integrations
+##### EWS O365
+Added a link in the documentation to the instructions for setting the required permissions for a self-deployed app.
+
+##### O365 - Security And Compliance - Content Search (beta)
+- Command 'o365-sc-new-search', Argument 'search_name' will be optional if empty will auto generate name with the following pattern (`XSOAR-<GUID>').
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.1.0.md:927]:
```suggestion
- Command 'o365-sc-new-search', Argument 'search_name' will be optional if empty will auto generate name with the following pattern ('XSOAR-<GUID>').
```

File: content-repo/extra-docs/releases/21.1.0.md [hunk 2]
```diff
@@ -0,0 +1,1472 @@
...
+
+---
+
+### EWS Pack v1.6.1
+#### Integrations
+##### EWS O365
+Added a link in the documentation to the instructions for setting the required permissions for a self-deployed app.
+
+##### O365 - Security And Compliance - Content Search (beta)
+- Command 'o365-sc-new-search', Argument 'search_name' will be optional if empty will auto generate name with the following pattern ('XSOAR-<GUID>').
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.1.0.md:927]:
```suggestion
- Command 'o365-sc-new-search', Argument 'search_name' will be optional if empty will auto generate name with the following pattern ('XSOAR-\<GUID\>').
```

File: content-repo/extra-docs/releases/21.1.0.md [hunk 3]
```diff
@@ -0,0 +1,1472 @@
...
+
+##### Expanse Type
+
+#### Integrations
+##### Expanse v2
+The Expanse v2 integration for Cortex XSOAR leverages the Expander API to create incidents from Expanse issues. It also leverages Expanse's unparalleled view of the Internet to enrich IPs, domains and certificates using information from assets discovered by Expanse Expander and risky flows detected by Expanse Behavior.
+##### Expanse Expander Feed
+Use this feed to retrieve the discovered IPs/Domains/Certificates from Expanse Expander asset database.
+#### Layouts
+- ** - None**
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.1.0.md:152]: please fix this

**Comment by ShirleyDenkberg** [comment]
@Itay4  Doc review completed.

**Review by Itay4 — APPROVED** [review]

**Pull request merged**